lineage/Analyze.py

```diff
@@ -63,7 +63,10 @@ def run_Analyze_over(list_of_populations, num_states, parallel=True, **kwargs):
 
         prom_holder = []
         for idx, population in enumerate(list_of_populations):
```

Comment:
Oh wait... shouldn't this if statement be outside of this loop?!

Reply:
We mostly give lists to the `run_Analyze_over`.

Reply:
Oh right... lists of different state numbers for example... ok, let me take over this branch. I need to adjust it to see what's wrong.

Reply:
Ok, thank you very much.